frontend/src/components/ui/combobox.tsx

```diff
@@ -187,7 +190,7 @@ export const Combobox = <TValue,>({
               rootClassName={"px-1 h-8"}
               autoFocus={true}
               value={search}
-              onValueChange={onSearchChange}
+              onValueChange={customFilterFn ?? onSearchChange}
```

Comment:
we can probably re-use the `onSearchChange` above - and remove the need for `customFilterFn`